### PR TITLE
Assign seweranhas and sewer snakes to their own newly created factions

### PR DESCRIPTION
**Pull request by Night-Pryanik**
#### Summary
None

#### Purpose of change
Seweranhas and sewer snakes were fighting and killing each other.

#### Describe the solution
Created two new monster factions. Assigned monsters to them.

#### Describe alternatives you've considered
None.

#### Testing
Debug-spawned to underground level of island prison type b. Watched seweranhas and sewer snakes don't fight each other.

#### Additional context
None.

### CODE REVIEW

File: data/json/monster_factions.json [hunk 1]
```diff
@@ -431,5 +431,19 @@
...
+    "base_faction": "mutant",
+    "hate": "fish",
+    "neutral": "seweranha"
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "sewer_snake",
+    "base_faction": "mutant",
+    "hate": "fish",
+    "neutral": "sewer_snake"
```
Comment (actual-nh) [inline data/json/monster_factions.json:447]:
```suggestion
    "neutral": "seweranha"
  },
  {
    "type": "MONSTER_FACTION",
    "name": "sewer_snake",
    "base_faction": "mutant",
    "neutral": "sewer_snake"
```

Alternatively, fish need to be something other than neutral toward these two; I can push a PR to your branch to do that and some cleanup (grouping all mutant piscivores into a base faction with subfactions, so we can just then tell fish to not be neutral toward mutant_piscivores). Which do you prefer? @Venera3, any comments?

Comment (Night-Pryanik) [reply data/json/monster_factions.json:447]: If you wish fish not be neutral towards these two new factions, then how? Friendly? Hate? By mood?

Comment (actual-nh) [reply data/json/monster_factions.json:447]: Probably the same way they are toward the other two mutant factions that eat them (gator and crayfish) - hate.

Comment (actual-nh) [reply data/json/monster_factions.json:447]: Whoops, crayfish are insect faction, not mutant - leaving out of new "mutant_piscivores" proposed faction (working on PR to your branch).

Comment (Night-Pryanik) [reply data/json/monster_factions.json]: Ok, applied, thanks.

**Review by Maleclypse — APPROVED** [review]

**Pull request merged**